projects/igniteui-angular/schematics/utils/dependency-handler.ts

```diff
@@ -148,15 +160,25 @@ function includeDependencies(pkgJson: any, context: SchematicContext, tree: Tree
                 const workspace = getWorkspace(tree);
                 const project = workspace.projects[workspace.defaultProject];
                 const projectOptions = getTargetedProjectOptions(project, 'build');
-                const mainTsPath = getConfigFile(project, 'main');
+                const projectTestOptions = getTargetedProjectOptions(project, 'test');
+                const mainTsPath = getConfigFile(project, 'main', 'build');
+                const testTsPath = getConfigFile(project, 'main', 'test');
                 const hammerImport = 'import \'hammerjs\';\n';
                 const mainTsContent = tree.read(mainTsPath).toString();
+                const testTsContent = tree.read(testTsPath).toString();
                 // if there are no elements in the architect.build.options.scripts array that contain hammerjs
                 // and main.ts does not contain an import with hammerjs
                 if (!projectOptions.scripts.some(el => el.includes('hammerjs')) && !mainTsContent.includes(hammerImport)) {
                     // import hammerjs in the main.ts file
-                    const contents = hammerImport + mainTsContent;
-                    tree.overwrite(mainTsPath, contents);
+                    const mainContents = hammerImport + mainTsContent;
+                    tree.overwrite(mainTsPath, mainContents);
+                }
+                // make sure test.ts has hammerjs import even for projects already containing it in main.ts
```

Comment:
Since there is a lot of repetition here, I'd extract this in a separate function, something like (name pending):
```suggestion
function addHammerToConfig(project: WorkspaceProject<ProjectType>, tree: Tree, config: string) {
    const projectOptions = getTargetedProjectOptions(project, config);
    const tsPath = getConfigFile(project, 'main', config);
    const hammerImport = 'import \'hammerjs\';\n';
    const tsContent = tree.read(tsPath).toString();
    // if there are no elements in the architect[config]options.scripts array that contain hammerjs
    // and the "main" file does not contain an import with hammerjs
    if (!projectOptions.scripts.some(el => el.includes('hammerjs')) && !tsContent.includes(hammerImport)) {
        // import hammerjs in the main file
        const mainContents = hammerImport + tsContent;
        tree.overwrite(tsPath, mainContents);
    }
}
```
Might be a bit too much, but it's there's more granularity this way. Thoughts?